gammapy/datasets/spectrum.py

```diff
@@ -325,15 +327,28 @@ def read(cls, filename, format="ogip", checksum=False, name=None, **kwargs):
         """
         from .io import OGIPDatasetReader
 
-        if format == "gadf":
-            return super().read(
-                filename, format="gadf", checksum=checksum, name=name, **kwargs
-            )
+        if format is None:
+            with fits.open(filename) as hdulist:
```

Comment:
Note that this won't work in general (e.g. env variable). You should use `path = make_path(filename)`

This is why tests fail.

Reply:
I believe this is fixed now